draft-ietf-httpbis-http2bis.xml

```diff
@@ -263,8 +263,9 @@
       </t>
       <t>
         The means by which support for HTTP/2 is determined is different for "http" and "https"
-        URIs. Discovery for "http" URIs is described in <xref target="discover-http"/>.  Discovery
-        for "https" URIs is described in <xref target="discover-https"/>.
+        URIs.  Discovery for "https" URIs is described in <xref target="discover-https"/>. HTTP/2
```

Comment:
Probably something for a different change, but "discovery" isn't really right.  The use of HTTP/2 for "https" is negotiated.